test/unit/utility/math_test.cpp

```diff
@@ -36,8 +36,8 @@ TYPED_TEST(unsigned_operations, floor_log2)
     for (uint8_t log2_value = 0; log2_value < seqan3::detail::bits_of<unsigned_t>; ++log2_value)
     {
         unsigned_t start = unsigned_t{1u} << log2_value;
-        unsigned_t end = start << 1u;
-        for (unsigned_t n = start, k = 0u; n < end && k < max_iterations; ++n, ++k)
+        unsigned_t end = static_cast<unsigned_t>(std::min<size_t>(start / 2u, max_iterations));
+        for (unsigned_t n = start; n < end; ++n)
```

Comment:
There seems to be something wrong here.
end = start * 2
but also the result of std::min seems wrong:
I expeected something like:
`std::min(start, max_iterations) + start`

Reply:
Yes, that was very wrong 😆